scss/tech.scss

```diff
@@ -1,41 +1,78 @@
 .tech-background {
-    margin-left: 30%;
     background-clip: content-box;
     background-color: $white;
-    border-radius: 25px;
-    border: 7px solid $green;
+    border-radius: 5px;
+    border: 1px solid $green;
+    margin: 40px 0 10px 42px;
+}
+
+.modify-img {
+    padding-top: 5px;
 }
 
 .size-tech {
-    height: 1600px;
-    width: 1000px;
-    .modify {
-        height: 1400px;
-    }
+    height: 340px;
+    width: 220px;
+}
+
+.modify {
+    height: 400px;
+}
+
+.subtitle {
+    padding-bottom: 15px;
 }
 
 .tech {
-    margin-top: 40px;
-    margin-left: 40px;
+    margin: 10px 0 0 10px;
     background: $green;
-    border-radius: 25px;
+    border-radius: 5px;
 }
 
 .order-tech {
     display: flex;
     flex-direction: column;
-    justify-content: space-evenly;
+    justify-content: space-around;
```

Comment:
What if I expand this div? I think you should not use 'space-around', use 'center' and add margins instead. In that way if you expand the divs they will stay together.